src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql

```diff
@@ -68,28 +68,28 @@ BEGIN
-    (NEWID(),'99998', 'Aimee', 'Mcdonald', 'fpiyushpandey8115@traclecfa.ml', '+44 1632 960294',@now, @emptyGuid),
-    (NEWID(),'99999', 'Sophia', 'Rojas', 'csheikha@sporbaydar.cf', '+44 1632 960131',@now, @emptyGuid);
+    (NEWID(),'100000', 'Tim',      'Teabag',    'tim@test.test',                '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100001', 'Kim',      'Butcher',   'kimben@test.test',             '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100002', 'Tracy',    'Burton',    'tracy@test.test',              '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100003', 'Mason',    'Sponge',    'masponge@test.test',           '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100004', 'Ashley',   'Banjo',     'ashleray@test.test',           '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100005', 'Ethan',    'Burton',    'ethburt@test.test',            '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100006', 'Sienna',   'Hunt',      'sienna@test.test',             '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100007', 'Nick',     'Rocks',     'nickrock@test.test',           '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100008', 'Muhammad', 'Rogers',    'muhaers@rogers.test',          '+44(0)1234 567891',@now, @emptyGuid),
+    (NEWID(),'100009', 'Mohammed', 'Burton',    'mohameid@burton.test',         '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100010', 'Becky',    'Banjo',     'beck@beckcox.test',            '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100011', 'Jennifer', 'Sponge',    'jennnt@test.test',             '01234 567891',     @now, @emptyGuid),
+    (NEWID(),'100012', 'Oscar',    'Banjo',     'oscarrrds@banjo.test',         '+44(0)1234 567891',@now, @emptyGuid),
+    (NEWID(),'100013', 'Jeannie',  'Houdini',   'pahmed@test.test',             '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100014', 'Marisa',   'Fletcher',  'marflet@test.test',            '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100015', 'Myra',     'Samosa',    'myra280s@test.test',           '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100016', 'Renato',   'Samosa',    'xpamela.gomes.pa2@test.test',  '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100017', 'Angel',    'Teabag',    'nael@test.test',               '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100018', 'Lindsay',  'Burton',    'linburt@test.test',            '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100019', 'Xavier',   'Eggbert',   'esamuel@eggs.test',            '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100020', 'Mario',    'Rocks',     'noulmes14b@mario.test',        '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'99998',  'Aimee',    'Mcdonald', 'mcaimee@test.test',             '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'99999',  'Sophia',   'Rojas',    'rosa@test.test',                '+44 1234 567891',  @now, @emptyGuid);
```

Comment:
I think we probably need to tweak some of these names a little more. 'Ashley Banjo', even though it's a slightly unusual name, is a real person, for example.

Feel free to reuse the names used in `InsertSolutions`.

Reply:
I have made some changes and also reused names from `InsertSolutions`